### DISCUSSION TITLE
Information on version visible to user

### DISCUSSION THREAD

**Discussion by hallinpihlatie** [INSPIRE-MIF/helpdesk-validator#1035] [Q&A] [answered]
Is there a way to see which version of the validator (eg. 2024.1) that has been istalled when using the Docker installation? If not, would it be possilbe to add the version information so that it would be accessible through the user interface? It could be in the HTML source and doesn't need to be visible, so a commented out string in the source or similar would work. It would be helpful to ensure that the user knows which version is currenlty running.

**Comment by DanielMartinPerezdeLeon** [comment]
Dear @hallinpihlatie ,

thanks for your suggestion.

The current version of the validator is specified during the Dockerfile execution:

docker run --name inspire-validator -d -p 8090:8090 -v ~/etf:/etf [docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2024.0](https://docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2024.0)

Indeed, you are correct in noting that users accessing an instance of the validator have no immediate means of identifying the version in use. We are considering displaying this information on the web interface. Your input on this matter, as well as suggestions regarding its placement, would be greatly appreciated.

Best regards,

**Comment by jenriquesoriano** [comment, marked as answer]
Dear @hallinpihlatie , 

for the time being we can do what you suggest, i.e. include the version in a commented html string.
Thanks for your contribution.
It will be incorporated in the next release v2024.0.1.

Best regards,

**Reply by hallinpihlatie** [reply]
That's great. Many thanks.

**Reply by DanielMartinPerezdeLeon** [reply]
Dear @hallinpihlatie,

just wanted to notify that we have added this behaviour in the new release and that it can already be checked in the [VALIDATOR](https://inspire.ec.europa.eu/validator/home/index.html)

![image](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/114756164/ca960e90-55cd-4011-a282-66e44fb26452)

thanks for your contribution and advices!

Regards,

**Reply by ZakarFin** [reply]
Tested updating the container in our env and the version can now be found on the HTML-source 👍

**Reply by hallinpihlatie** [reply]
Yes, thanks
